config/urls.py

```diff
@@ -1,8 +1,14 @@
 """Project URL Configuration"""
 from django.contrib import admin
 from django.urls import path, include
+from django.conf import settings
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', include('quran.urls')),
 ]
+
+if settings.DEBUG:
+    from django.conf.urls.static import static
```

Comment:
Tbh, I don't like this kind of imports :)
I prefer moving everything to the top for visibility.

Reply:
Don't think this import will have any influence on prod, so no problem.